magenta/music/musicxml_parser.py

```diff
@@ -544,6 +545,31 @@ def _parse_forward(self, xml_forward):
                * self.state.seconds_per_quarter)
     self.state.time_position += seconds
 
+    # Issue #674 - If the <forward> element is in a measure without
+    # any <note> elements, treat it as if it were a whole measure
+    # rest by inserting a rest of that duration
+    note_count = len(self.xml_measure.findall('./note'))
```

Comment:
Good idea. That simplifies the code. I moved this code into the Part class, into a new method called `_repair_empty_measure`. This method is immediately called before a measure is parsed.